src/types.ts

```diff
@@ -0,0 +1,352 @@
+export interface User {
+    login: string;
+    id: number;
+    avatar_url: string;
+    gravatar_id: string;
+    url: string;
+    html_url: string;
+    followers_url: string;
+    following_url: string;
+    gists_url: string;
+    starred_url: string;
+    subscriptions_url: string;
+    organizations_url: string;
+    repos_url: string;
+    events_url: string;
+    received_events_url: string;
+    type: string;
+    site_admin: boolean;
+}
+
+export interface Label {
+    id: number;
+    url: string;
+    name: string;
+    color: string;
+    default: boolean;
+}
+
+export interface Assignee {
+    login: string;
+    id: number;
+    avatar_url: string;
+    gravatar_id: string;
+    url: string;
+    html_url: string;
+    followers_url: string;
+    following_url: string;
+    gists_url: string;
+    starred_url: string;
+    subscriptions_url: string;
+    organizations_url: string;
+    repos_url: string;
+    events_url: string;
+    received_events_url: string;
+    type: string;
+    site_admin: boolean;
+}
+
+export interface Creator {
+    login: string;
+    id: number;
+    avatar_url: string;
+    gravatar_id: string;
+    url: string;
+    html_url: string;
+    followers_url: string;
+    following_url: string;
+    gists_url: string;
+    starred_url: string;
+    subscriptions_url: string;
+    organizations_url: string;
+    repos_url: string;
+    events_url: string;
+    received_events_url: string;
+    type: string;
+    site_admin: boolean;
+}
+
+export interface Milestone {
+    url: string;
+    html_url: string;
+    labels_url: string;
+    id: number;
+    number: number;
+    state: string;
+    title: string;
+    description: string;
+    creator: Creator;
+    open_issues: number;
+    closed_issues: number;
+    created_at: Date;
+    updated_at: Date;
+    closed_at: Date;
+    due_on: Date;
+}
+
+export interface PullRequest {
+    url: string;
+    html_url: string;
+    diff_url: string;
+    patch_url: string;
+}
+
+export interface Owner {
+    login: string;
+    id: number;
+    avatar_url: string;
+    gravatar_id: string;
+    url: string;
+    html_url: string;
+    followers_url: string;
+    following_url: string;
+    gists_url: string;
+    starred_url: string;
+    subscriptions_url: string;
+    organizations_url: string;
+    repos_url: string;
+    events_url: string;
+    received_events_url: string;
+    type: string;
+    site_admin: boolean;
+}
+
+export interface Permissions {
+    admin: boolean;
+    push: boolean;
+    pull: boolean;
+}
+
+export interface Repository {
```

Comment:
With the exception of the bloat in this merge, what's the downside of leaving them?